app/experimenter/kinto/tests/test_tasks.py

```diff
@@ -442,19 +449,21 @@ def test_check_experiment_that_should_pause_does_pause(
 class TestCheckExperimentIsLive(MockKintoClientMixin, TestCase):
     def test_experiment_updates_when_record_is_in_main(self):
         experiment1 = NimbusExperimentFactory.create_with_status(
-            NimbusExperiment.Status.ACCEPTED,
+            NimbusExperiment.Status.DRAFT,
+            publish_status=NimbusExperiment.PublishStatus.WAITING,
         )
 
         experiment2 = NimbusExperimentFactory.create_with_status(
-            NimbusExperiment.Status.ACCEPTED,
+            NimbusExperiment.Status.DRAFT,
+            publish_status=NimbusExperiment.PublishStatus.WAITING,
         )
 
         experiment3 = NimbusExperimentFactory.create_with_status(
             NimbusExperiment.Status.DRAFT,
         )
 
-        self.assertEqual(experiment1.changes.count(), 3)
-        self.assertEqual(experiment2.changes.count(), 3)
+        self.assertEqual(experiment1.changes.count(), 1)
+        self.assertEqual(experiment2.changes.count(), 1)
         self.assertEqual(experiment3.changes.count(), 1)
```

Comment:
Kinda wonder if these assertions are really that important now?

Reply:
Eh yeah it was just a way of showing that the new assertions show up after the task.  Could rewrite them or just take them out, not that important I don't think.